### ISSUE TITLE
clean up after a run

### ISSUE THREAD

**Issue by bstabler** [RSGInc/SOABM#32]
delete files such as earlier CT-RAMP iteration outputs, intermediate truck trip matrices, parking cost debug files, etc.  Maybe just delete the entire "other" folder?  Let's plan to do this after @bettinardi reviews the next iteration of the model setup.

**Comment by bettinardi** [comment]
This is related to the new issue I created #50.
We should revisit this and clean-up a lot of these run issues before getting into the final stage of calibration.  I think there are enough side issues to cast doubt that the model is actually running the way we think it is.

Side issues being:
Model takes too long to setup
Not sure which transit checks to run, not sure if transit walk access / lines are correct.
Not sure about truck PCE vs volumes
Need the model run to critical fail when a piece (likely Visum) critically fails (see SWIM error checking)
As this issue calls out - we need a cleaner end state.
Not clear that we have implemented TypeNo correctly (currently all zero, one vdf, but seems that there should be more than one).
